src/Core/src/Core/IFrameworkElement.cs

```diff
@@ -8,7 +8,7 @@ namespace Microsoft.Maui
 	/// <summary>
 	/// Represents a framework-level set of properties, events, and methods for .NET MAUI elements. 
 	/// </summary>
-	public interface IFrameworkElement
+	public interface IFrameworkElement : ITransform
```

Comment:
This should be on `IView` as we can't transform non-views.

Reply:
I think this is fine for right now because right now

IFrameworkElement == VisualElement on the Controls side

and VisualElement is what we have all our TRansform APIs on in Controls

So once we get a better separation of interfaces then we can set this up better?

Reply:
Yes, we will be able to move it.